.github/workflows/examples-efr32.yaml

```diff
@@ -95,13 +95,18 @@ jobs:
                     'import("//with_pw_rpc.gni")'
                   .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py efr32 BRD4161A+rpc lighting-app \
                     out/lighting_app_debug_rpc/BRD4161A/chip-efr32-lighting-example.out /tmp/bloat_reports/
-            # Disabled until Flash usage is reduced        
-            # - name: Build example EFR32+WF200 WiFi Lock app for BRD4161A
-            #   timeout-minutes: 10
-            #   run: |
-            #       scripts/examples/gn_efr32_example.sh examples/lock-app/efr32/ out/lock_app_wifi_wf200 BRD4161A --wifi wf200
-            #       .environment/pigweed-venv/bin/python3 scripts/tools/memory/gh_sizes.py efr32 BRD4161A+wf200 lock-app \
-            #         out/lock_app_wifi_wf200/BRD4161A/chip-efr32-lock-example.out /tmp/bloat_reports/
+            - name: Build example EFR32+WF200 WiFi Lock app for BRD4161A
+              timeout-minutes: 10
+              run: |
+                  scripts/examples/gn_efr32_example.sh examples/lock-app/efr32/ out/lock_app_wifi_wf200 BRD4161A is_debug=false --wifi wf200
```

Comment:
can build_examples.py be used instead?

I would prefer more uniform build instructions and build_examples.py is the uniform interface.

Reply:
@andy31415 this can be tackled in a different PR.